json_formatter.go

```diff
@@ -68,8 +68,24 @@ func (f *JSONFormatter) Format(entry *Entry) ([]byte, error) {
 			// Otherwise errors are ignored by `encoding/json`
 			// https://github.com/sirupsen/logrus/issues/137
 			data[k] = v.Error()
-		default:
+		case string:
 			data[k] = v
+		case bool:
+			data[k] = bool(v)
+		case int, int8, int16, int32, int64:
+			data[k] = v.(int)
+		case uint, uint8, uint16, uint32, uint64:
```

Comment:
just to make sure, running v.(uint) on a uint64 is giving the same result?
usually, uint (without any suffix is uint32). wondering if it will cut off the value..